Lib/test/test_typing.py

```diff
@@ -1422,6 +1422,17 @@ class CustomProtocol(TestCase, Protocol):
         class CustomContextManager(typing.ContextManager, Protocol):
             pass
 
+    def test_non_runtime_protocol_isinstance_check(self):
+        class P(Protocol):
+            pass
+
+        class A:
+            pass
+
+        with self.assertRaises(TypeError):
+            isinstance(A(), P)
```

Comment:
I think we can simplify it down even more, also I think ``assertRaisesRegex`` (https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaisesRegex) is better, because ``isinstance`` can throw ``TypeError`` for many reasons:
```suggestion
        with self.assertRaisesRegex(TypeError, "@runtime_checkable"):
            isinstance(1, P)
```

Reply:
Thanks, code looks much simpler)